server/views.py

```diff
@@ -1,5 +1,38 @@
-from django.shortcuts import render
-
+from django.shortcuts import render, redirect
+from .forms import UserForm, PlayerForm, MembershipForm, VaccinationForm
+from .models import User
 
 def home(request):
-    return render(request, "index.html")
+    return render(request, 'index.html')
```

Comment:
Can you use [`black`](https://github.com/psf/black) to format the code?

Reply:
added `black` formatting to `server/`